ViewLayer/DebugDisplay/MotorPage/MotorUi/MotorView.cpp

```diff
@@ -0,0 +1,374 @@
+#include "MotorView.h"
+
+MotorView::MotorView(KeyMotorPresenter& keyMotorPresenter, MotorDetailsPresenter& motorDetailsPresenter,
+                         I_MotorUi& ui)
+    : keyMotorPresenter_(keyMotorPresenter)
+    , motorDetailsPresenter_(motorDetailsPresenter)
+    , ui_(ui)
+{
+    connectMotor(keyMotorPresenter_, motorDetailsPresenter_ );
+}
+
+MotorView::~MotorView()
+{
+}
+
+void MotorView::connectMotor(KeyMotorPresenter& keyMotorPresenter, MotorDetailsPresenter& motorDetailsPresenter)
+{
+    connect(&keyMotorPresenter, SIGNAL(motorZeroReceived(KeyMotor)),
+            this, SIGNAL(motorZeroReceived(KeyMotor)));
+    connect(&keyMotorPresenter, SIGNAL(motorOneReceived(KeyMotor)),
+            this, SIGNAL(motorOneReceived(KeyMotor)));
+    connect(&keyMotorPresenter, SIGNAL(motorSetCurrentReceived(double setCurrent)),
```

Comment:
For signals, let's keep things consistent and do `SIGNAL(motorSetCurrentReceived(double)`
Actually I think we need to do it like that, or else the signals won't work properly.
This comment goes for the rest of them.

Reply:
This comment needs to be addressed